docs/dev.md

```diff
@@ -1,84 +1,106 @@
 # Development
 
-You'll need to use Nightly and about:debugging to use unsigned development
-versions of the add-on.
+There are two variants of Normandy Devtools: the extension and the website. The
+extension has super powers because it can integrate deeply into Firefox. The
+website is more accessible and a little easier to work on.
 
-> WARNING: This workflow is only useful for working on the extension itself.
-> If you aren't working on the code yourself, you should get a release build
-> from above.
+## Setup
 
-```
+Clone the repository, and install the dependencies:
+
+```sh
 $ git clone https://github.com/mozilla/normandy-devtools.git
 $ cd normandy-devtools
 $ yarn install
 ```
 
-To run the extension in development mode, you can simply use the `watch` command to
+## Extension
+
+To work on the extension version NDT, you'll need to have Firefox Nightly.
+
+> WARNING: This workflow is only useful for working on the extension itself. If
+> you aren't working on the code yourself, you should get a release build from
+> [the releases page on Github][releases]. Non-development builds of NDT involve
+> a signing process not covered here.
+
+To run the extension in development mode, you can use the `watch-ext` command to
 build the extension, package it and install it into a Nightly temporary profile:
 
+```sh
+$ yarn watch-ext
 ```
-$ yarn watch
-```
 
-> NOTE: If you are seeing an error while trying to run `watch` make sure that Nightly
-> is installed on your system and if you are on Linux make sure that `firefox-nightly`
-> is on your PATH.
+> NOTE: If you are seeing an error while trying to run `watch-ext` make sure
+> that Nightly is installed on your system. If you are on Linux check that
+> `firefox-nightly` is on your PATH.
+
+## Web
```

Comment:
*nipick* Above we said website, below webpage. They are obvious synonyms but I think wee could stick to one same word, no?